rootfs/Dockerfile

```diff
@@ -3,13 +3,14 @@ FROM gcr.io/google_containers/ubuntu-slim:0.4
 ENV TERM=xterm
 
 # disable source repos (speeds up apt-get update)
-RUN sed -i -e 's/^deb-src/#deb-src/' /etc/apt/sources.list && \
-apt-get update && \
-apt-get install -y \
-	bash \
-	curl \
-	net-tools \
-	procps && \
-	apt-get clean && \
-	rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/* /usr/share/man /usr/share/doc && \
-bash -c "mkdir -p /usr/share/man/man{1..8}"
+RUN sed -i -e 's/^deb-src/#deb-src/' /etc/apt/sources.list \
+	&& apt-get update \
+	&& apt-get upgrade -y \
+	&& apt-get install -y \
```

Comment:
Could you change this to:
```Dockerfile
        && DEBIAN_FRONTEND=noninteractive apt-get install -y \
```
and the same for the `apt-get upgrade` command? I find those warnings about "no terminal found" distracting.